com/stencyl/models/GameModel.hx

```diff
@@ -65,11 +67,11 @@ class GameModel
 		actualWidth = Std.parseInt(xml.att.awidth);
 		actualHeight = Std.parseInt(xml.att.aheight);
 		scale = Std.parseInt(xml.att.scale);
-		defaultSceneID = 0;
-		
+		defaultSceneID = IdUtils.INVALID_ID;
```

Comment:
Yes, it's not, but 0 is also wrong, I guess?

If I create a game, add two scenes, then delete the first one 0 is not a valid scene either...